src/models/unit_tests/rule_test.rs

```diff
@@ -154,7 +154,7 @@ fn test_get_edit_negative_recursive() {
       ]
   };
   let source_code = "class Test {
-          pub void foobar(){
```

Comment:
It seems that I had some syntactically incorrect test cases. I had to fix them because `allow-dirty-ast` is `false` by default